src/pages/Browse/index.tsx

```diff
@@ -159,11 +160,15 @@ export class BrowseBase extends React.Component<Props> {
         {file ? (
           <React.Fragment>
             <Col md="8">
-              <CodeView
-                mimeType={file.mimeType}
-                content={file.content}
-                version={version}
-              />
+              {file.type === 'image' ? (
+                <ImageView content={file.content} mimeType={file.mimeType} />
+              ) : (
+                <CodeView
+                  mimeType={file.mimeType}
+                  content={file.content}
+                  version={version}
+                />
+              )}
             </Col>
             <Col md="1">
               <CodeOverview content={file.content} version={version} />
```

Comment:
Funny enough, passing image content to `CodeOverview` totally works. We should probably not do that, though, since it could be confusing. I think you could pass it an empty string for the case of images to not show anything. If that doesn't work, file an issue.

Reply:
Passing an empty string works and the `CodeOverview` appears as empty.

Reply:
I think that ideally we'd also pass no `version`, and `CodeOverview` would be smart enough not to do all the message processing if no `version` were passed, but that could be a follow-up. Or maybe we render `null` instead of `CodeOverview` when `file.type === 'image'`?